pkg/analysis_server/test/services/correction/fix_test.dart

```diff
@@ -5915,6 +5915,92 @@ main() {
 ''');
   }
 
+  test_lint_replaceWithCollectionLiteral_linked_hash_map_with_generic() async {
+    String src = '''
+import 'dart:collection';
+
+final a = /*LINT*/new LinkedHashMap<int,int>();
```

Comment:
I am having problem with the tests that uses LinkedHashMap from dart.collection, the type it is not being solved when running the tests but it is solved when using it because we compile our own local SDK.

Someone knows how to solve this?